float64[3] acceleration # x y z
float64[3] orientation # x y z (from gyroscope)

float64 phi

# encoder values
float64[2] wheel_speeds # left, right
float64[2] hip_angles # left, right
float64 neck_angle

# servo values
float64 head_angle 
float64 grasper_angle